packages/manager/apps/pci-public-ip/src/components/list/AdditionalIPServiceAction.tsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from 'react';
+import {
+  ODS_THEME_COLOR_INTENT,
+  ODS_THEME_TYPOGRAPHY_SIZE,
+} from '@ovhcloud/ods-common-theming';
+import {
+  ODS_BUTTON_SIZE,
+  ODS_BUTTON_VARIANT,
+  ODS_TEXT_LEVEL,
+} from '@ovhcloud/ods-components';
+import { OsdsButton, OsdsText } from '@ovhcloud/ods-components/react';
+import { useTranslation } from 'react-i18next';
+import { useNavigation } from '@ovh-ux/manager-react-shell-client';
+
+export const AdditionalIPServiceAction = () => {
+  const { t } = useTranslation();
+  const [url, setUrl] = useState('');
+  const nav = useNavigation();
+
+  useEffect(() => {
+    nav
+      .getURL('dedicated', `#/ip?serviceName=_FAILOVER&page=1`, {})
```

Comment:
```suggestion
      .getURL('dedicated', `#/ip?serviceName=_FAILOVER`, {})
```
nitpick: page=1 probably not necessary

Reply:
trying to be iso prod, i dont now the consequences of removing it

Reply:
After test, there is no side effect if page param not assigned, i remove it finally